### PR TITLE
[DevOps] Scripts for Exporting Telemetry Data

### PR DESCRIPTION
**Pull request by ChathurindaRanasinghe**
**Description:**

- Add a shell script and K8s Job to download telemetry data from HAIC clusters.

**Related Issues (optional):**
- https://github.com/h2oai/h2o-ops/issues/543

### CODE REVIEW

File: scripts/Job.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  name: haic-telemetry-export
+spec:
+  template:
+    metadata:
+      name: haic-telemetry-export
+    spec:
+      restartPolicy: OnFailure
+      containers:
+      - name: haic-telemetry-export-runtime
+        image: gcr.io/vorvan/h2oai/haic-telemetry-exporter:latest
```
Comment (mmalohlava) [inline scripts/Job.yaml:13]: @ChathurindaRanasinghe QQ: where is the Dockefile for the container?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# HAIC Telemetry Export
+
+## User Guide
+
+### Prerequisites
+
+- Make sure you have following installed
+    - kubectl
+    - AWS CLI
```
Comment (mmalohlava) [inline README.md:9]: We need to execute it on-prem - no AWS.

File: scripts/Job.yaml (hunk 1 above)
Comment (ChathurindaRanasinghe) [reply scripts/Job.yaml:13]: Here @mmalohlava https://github.com/h2oai/cloud-utils/pull/35/files#diff-313bf94e58b04ca85b029fe151ffec6bd19167331cec7dbf66af017ded5e9654

File: README.md (hunk 2 above)
Comment (ChathurindaRanasinghe) [reply README.md:9]: Sure @mmalohlava  will look into it. Here AWS CLI is needed only to get the access to the cluster (update the `kubeconfig`) once it is done kubectl is used.

File: scripts/Job.yaml (hunk 1 above)
Comment (mmalohlava) [reply scripts/Job.yaml:13]: thank you! pls document here in this repo as well if possible.

File: scripts/export_telemetry.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+awk -v db_dsn="$TELMETRY_DB_DSN" '{gsub("_SUB_DB_DSN_", db_dsn)} 1' scripts/Job.yaml > temp && mv temp scripts/Job.yaml
+
+echo "Starting job ..."
+kubectl apply -f scripts/Job.yaml --namespace=$NAMESPACE 
+
+echo "Waiting 120s until pod starts up ..."
+sleep 120
+
+pod=$(kubectl get pods -n telemetry --output=jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}' | grep haic-telemetry-export)
```
Comment (mmalohlava) [inline scripts/export_telemetry.sh:20]: @hasithjp should it use `NAMESPACE` variable ?

**Pull request merged**